symfony/plugins/orangehrmAttendancePlugin/Service/AttendanceService.php

```diff
@@ -90,7 +90,7 @@ public function savePunchRecord(AttendanceRecord $attendanceRecord) {
      */
     public function getLastPunchRecord($employeeId, $actionableStatesList) {
```

Comment:
do we need this service method? since it directly calling DAO method

Reply:
done